angularSTEP/src/app/profile-menu/profile-menu.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-profile-menu',
+  templateUrl: './profile-menu.component.html',
+  styleUrls: ['./profile-menu.component.scss']
+})
+export class ProfileMenuComponent implements OnInit {
+
+  @Input() user: any;
+  username: string;
+  displayName: string;
+  picUrl: string;
+  //fixed value for now - implement input to show if user is actually signed in
+  loggedIn: boolean;
+
+  constructor() {}
+
+  ngOnInit(): void {
+    this.displayName = this.user.displayName;
+    this.username = this.user.username;
+    this.picUrl = this.user.picUrl;
+    this.loggedIn = this.user.loggedIn;
+  }
+
+  //temporary logout and signin functions
+  logout(){
+    this.loggedIn = !this.loggedIn;
+  }
+
+  signIn(){
```

Comment:
Again I'm not 100% sure on styling, but based off the other functions in this file there should be a space between the close parenthesis and the open brace